cli/lib/errors.js

```diff
@@ -2,5 +2,6 @@
 
 module.exports = {
   CouldNotReadConfigFile: class extends Error {},
-  CouldNotParseConfigToml: class extends Error {}
```

Comment:
I am using this error message in this PR https://github.com/entropic-dev/entropic/pull/186

Reply:
The error isn't removed. See the code below this line